framework/include/problems/FEProblemBase.h

```diff
@@ -1294,22 +1295,29 @@ class FEProblemBase : public SubProblem, public Restartable
   ///@{
   /**
    * These methods are used to determine whether stateful material properties need to be stored on
-   * internal sides.  There are four situations where this may be the case: 1) DGKernels
-   * 2) IntegratedBCs 3)InternalSideUserObjects 4)ElementalAuxBCs
+   * internal sides.  There are five situations where this may be the case: 1) DGKernels
```

Comment:
I would argue that there are 6; one could have InterfaceKernels without InterfaceUserObjects just as one can have `DGKernels` without `InternalSideUserObjects`